components/faqs/nesting.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { withTheme, ThemeProvider } from 'styled-components'
+
+import SectionLayout from '../SectionLayout'
+import CodeBlock from '../CodeBlock'
+import LiveEdit from '../LiveEdit'
+
+const scope = { withTheme, ThemeProvider }
+const sample = (`
+const ColorChanger = styled.section\`
+  background: papayawhip;
+
+  > h2 {
+    color: palevioletred;
+  }
+
+  @media(min-width: 768px) {
+    background: mediumseagreen;
+
+    > h2 {
+      color: papayawhip;
+    }
+  }
+\`;
+render(
+  <ColorChanger href="#">
+    <h2>Hello world!</h2>
+  </ColorChanger>
+)
+`).trim()
+
+const Nesting = () => (
+  <SectionLayout title="Nesting">
```

Comment:
@oliverturner Last thing :) Can we change the titles to questions? I'd expect that it feels more natural under an FAQ section